rwth_upper_body_skeleton_random_walk/package.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<package>
+  <name>rwth_upper_body_skeleton_random_walk</name>
+  <version>0.0.0</version>
+  <description>The upper_body_skeleton package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag --> 
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="mmp@todo.todo">mmp</maintainer>
```

Comment:
This line needs fixing to include the *real* maintainer, including a valid email before merging.